applications/fishing-map/src/features/datasets/NewDataset.tsx

```diff
@@ -80,36 +113,69 @@ const DatasetConfig: React.FC<DatasetConfigProps> = (props) => {
           console.log('removed', removed)
         }}
         direction="top"
-      />
+      /> */}
       <div className={styles.row}>
         <label className={styles.selectLabel}>
           {t('dataset.colorByValue', 'Color features by value')}
         </label>
         <Select
           className={styles.selectShort}
-          options={[]}
-          selectedOption={undefined}
+          containerClassName={styles.selectContainer}
+          options={colorByOptions}
+          selectedOption={colorByOptions.find(
+            ({ id }) => id === metadata.configuration?.propertyToInclude
+          )}
           onSelect={(selected) => {
-            console.log('selected', selected)
+            // TODO: pre-populate min and max values depending on selection
+            onDatasetFieldChange({ propertyToInclude: selected.id })
+            const propertyRange = getPropertyRangeValuesFromGeojson(fileData, selected.id)
+            if (propertyRange) {
+              onDatasetFieldChange({
+                propertyToIncludeRange: {
+                  min: propertyRange.min,
+                  max: propertyRange.max,
+                },
+              })
+            }
           }}
-          onRemove={(removed) => {
-            console.log('removed', removed)
+          onRemove={() => {
+            onDatasetFieldChange({ propertyToInclude: undefined })
           }}
           direction="top"
         />
         <InputText
           inputSize="small"
+          type="number"
+          step="0.1"
+          value={metadata.configuration?.propertyToIncludeRange?.min}
           placeholder={t('common.min', 'Min')}
           className={styles.shortInput}
-          onChange={(e) => console.log(e.target.value)}
+          onChange={(e) =>
+            onDatasetFieldChange({
+              propertyToIncludeRange: {
+                min: parseFloat(e.target.value),
```

Comment:
Where do we validate that the min is not greater than the max ?

Reply:
Done!
![image](https://user-images.githubusercontent.com/10500650/107884361-ebad2280-6ef4-11eb-808a-72b526a2f9fc.png)